src/me/drton/jmavsim/Simulator.java

```diff
@@ -306,7 +306,7 @@ private AbstractMulticopter buildMulticopter() {
         I.m11 = 0.005;  // Y
         I.m22 = 0.009;  // Z
         vehicle.setMomentOfInertia(I);
-        vehicle.setMass(1.0); // mass of the vehicle, too light the vehicle will start climbing on arm
+        vehicle.setMass(0.8);
```

Comment:
at 0.8kg, the Iris hovers at about 50% throttle. Increasing the mass to 1kg requires over 60% throttle to hover. No one has replied to the issue I raised: https://github.com/PX4/jMAVSim/issues/32